src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -343,6 +344,15 @@ CreateInsertSelectRouterPlan(Query *originalQuery,
 		++taskIdIndex;
 	}
 
+	if (list_length(sqlTaskList) > 1 && MultiTaskQueryLogLevel !=
```

Comment:
If we put `MultiTaskQueryLogLevel != MULTI_TASK_QUERY_INFO_OFF`, these would be more readable.

```
if (list_length(sqlTaskList) > 1 && 
MultiTaskQueryLogLevel != MULTI_TASK_QUERY_INFO_OFF)
```
```
if (MultiTaskQueryLogLevel != MULTI_TASK_QUERY_INFO_OFF && 
   (list_length(sqlTaskList) > 1
```